pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -414,6 +415,16 @@ static Value createInit(OpBuilder &builder, Location loc, Type type,
       auto value = convertFloatUsingType(llvm::APFloat(1.0), floatType);
       return builder.create<mlir::ConstantFloatOp>(loc, value, floatType);
     }
+    case AggregationKind::min: {
+      auto value = convertFloatUsingType(
+          llvm::APFloat(std::numeric_limits<float>::max()), floatType);
+      return builder.create<mlir::ConstantFloatOp>(loc, value, floatType);
+    }
+    case AggregationKind::max: {
+      auto value = convertFloatUsingType(
+          llvm::APFloat(std::numeric_limits<float>::min()), floatType);
```

Comment:
This should be the most negative value of a float, i.e. `-std::numeric_limits<float>::max()`. See https://stackoverflow.com/questions/3529394/obtain-minimum-negative-float-value-in-c and https://en.cppreference.com/w/cpp/types/climits

Reply:
Oops, good catch.

Reply:
Actually, we might want to use +- inf....

Reply:
That's nice because then the type (double/float) doesn't matter, since inf is inf regardless of precisison.